.gitignore

```diff
@@ -25,84 +25,35 @@ wheels/
 .installed.cfg
 *.egg
 
-# PyInstaller
-#  Usually these files are written by a python script from a template
-#  before PyInstaller builds the exe, so as to inject date/other infos into it.
-*.manifest
-*.spec
+
+# ---- Not in Use ----
```

Comment:
Don't need to delete them, this is a standard python git ignore

Reply:
Lol but these are the ignores we definitely won't ever use for Modmail.

Reply:
I Added them back according to https://github.com/github/gitignore/blob/master/Python.gitignore